docker/development/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM ubuntu:18.04
 ARG GO_VERSION=1.14.3
 ARG GO_PROTOC_VERSION=3.6.1
 ARG GO_PROTOC_GEN_GO_VERSION=1.21.0
-ARG GOLANGCILINT_VERSION=1.23.6
+ARG GOLANGCILINT_VERSION=1.28.2
```

Comment:
Would be nice to also install `gofumpt` into the dev docker image?

Reply:
Good idea, added.